src/bitte-wallet.ts

```diff
@@ -153,26 +152,6 @@ const signMessage = async (
   });
 };
 
-const completeTransaction = async (
```

Comment:
Turns out this is not needed for Bitte Wallet

Reply:
great